admin_ui/src/components/KeySearch.vue

```diff
@@ -79,7 +78,7 @@ export default {
                 return 0
             })
         },
-        selectResult(id, readable) {
+        selectResult(id: undefined, readable: undefined) {
```

Comment:
The type here looks a bit odd.

I'd try `selectResult(id: number | undefined, readable: boolean | undefined)`

Reply:
Unfortunately `selectResult(id: number | undefined, readable: boolean | undefined)` does not work. Only `selectResult(id: any | undefined, readable: any | undefined)` work.

Reply:
OK, let's just have it as `any` for now then. I think having `undefined` is a strange looking one, as if the argument was always of type undefined, it wouldn't be a very useful function!